server/controllers/recipesController.js

```diff
@@ -14,14 +14,14 @@ const suggestRecipes = async (req, res) => {
   try {
     // URL of N8n webhook per https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.webhook/?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link&utm_campaign=n8n-nodes-base.webhook
     //const webhookUrl = `${process.env.N8N_WEBHOOK_URL}/recommend-recipes`;
-    const webhookUrl = process.env.N8N_WEBHOOK_URL_RECIP_TEST;
+    const webhookUrl = process.env.N8N_WEBHOOK_URL_RECIPE;
```

Comment:
There is already existing N8N_WEBHOOK_URL in https://github.com/nicmart-dev/feedmenow/blob/develop/server/example.env
Not sure why we need this new one?